src/Plaid/PlaidOptions.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Going.Plaid
+{
+	/// <summary>
+	/// Implements the IOptions pattern for reading a 'Plaid' section from an IConfiguration object.
+	/// </summary>
+	public class PlaidOptions
+	{
+		/// <summary>
+		/// The section name in the IConfiguration that contains PlaidOptions settings
+		/// </summary>
+		public const string SectionKey = "Plaid";
+
+		/// <summary>
+		/// client_id is the same across all environments
+		/// !!! THIS SHOULD NEVER BE SHARED IN CLIENT-SIDE CODE !!!
+		/// See: https://plaid.com/docs/quickstart/#api-keys
```

Comment:
`<see href="https://..." />`.  Should also move this comment to the `<remarks>` tag instead of in the `<summary>` tag.